apps/dashboard/app/assets/javascripts/apps.js.coffee

```diff
@@ -1,3 +1,8 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+
+$ ->
+  $('#all-apps-table').DataTable({
+    stateSave: true
+  })
```

Comment:
Just to be clear - we don't want to support turbolinks here?

I get that _we_ don't currently use it, but you can imagine that someone has enabled it.  Or at least would if they knew it exists - using it while testing this out, it really speeds things up, it's not clear to me why we wouldn't want to use it at OSC or evangelize it's use.

Reply:
Turbolinks and jquery mostly don't play well together, as is evidenced by this discussion. However, there are too many examples of code in our code base or in the form.js files (that are now in the wild, in use by other people) that use something like `$(...)` or `$(document).ready(...)` which of course is in conflict with Turbolinks. Also, Turbolinks introduces accessibility issues that I'm not sure how to address yet.

Reply:
It is also superseded by https://github.com/hotwired/turbo and the old Turbolinks is no longer supported. I don't know the differences.